zipkin/src/main/java/zipkin/storage/QueryRequest.java

```diff
@@ -178,7 +178,7 @@ public static Builder builder() {
     private Long minDuration;
     private Long maxDuration;
     private Long endTs;
-    private Long lookback;
+    private Long lookback = 86400000L; // see ZipkinQueryApiV1.defaultLookback
```

Comment:
I've removed it, and added it to the tests in zipkin-storage/cassandra-3
But it'll be a performance problem for tests, as many of the tests are in SpanStoreTest and SpanCassandraTest.